ct/apprise-api.sh

```
@@ -0,0 +1,75 @@
+    exit
+  fi
+  if check_for_gh_release "apprise" "caronc/apprise-api"; then
+    msg_info "Stopping Service"
+    systemctl stop apprise-api
+    msg_ok "Stopped Service"
+
+    PYTHON_VERSION="3.12" setup_uv
+    CLEAN_INSTALL=1 fetch_and_deploy_gh_release "apprise" "caronc/apprise-api" "tarball"
+
+    msg_info "Updating Apprise-API"
+    cd /opt/apprise
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)